### PR TITLE
chore: Create changelog for past few weeks of changes

### LINKED ISSUE #60: Fill in changelog for next release
Changes: https://github.com/rust-windowing/softbuffer/compare/v0.1.1...master

I could use some help from others who have submitted PRs recently.

### PR DESCRIPTION
**Pull request by notgull**
Closes #60. Let me know if there's anything I missed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# v0.1.1
```
Comment (jackpot51) [inline CHANGELOG.md:1]: Make this `UNRELEASED`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+* Add support for Redox/Orbital.
+* Add support for BSD distributions.
+* Ported Windows backend from `winapi` to `windows-sys`.
+* **Breaking:** Take a reference to a window instead of owning the window.
+* Add a `from_raw` function for directly using raw handles.
+* **Breaking:** Add feature flags for `x11` and `wayland` backends.
+* Use static dispatch instead of dynamic dispatch for the backends.
+* Add `libxcb` support to the X11 backend.
```
Comment (jackpot51) [inline CHANGELOG.md:10]: Copy the lines from README.md for 0.1.1 and 0.1.0, then link to this file from the README.md

**Comment by ids1024** [comment]
The Wayland backend has been updated to wayland-client 0.30, and more importantly has been fixed to swap buffers without resulting in errors or visual corruption, and handle resizes smoothly.

**Comment by ids1024** [comment]
This also doesn't mention https://github.com/rust-windowing/softbuffer/pull/46, which is hopefully a notable performance gain.

**Comment by jackpot51** [comment]
I did add a few more notes before the release.

**Pull request merged**